internal/volumes/mount.go

```diff
@@ -134,7 +139,20 @@ func (s *LinuxMountService) Publish(targetPath string, devicePath string, opts M
 		return s.mounter.Mount(devicePath, targetPath, opts.FSType, mountOptions)
 	}
 
-	return s.mounter.FormatAndMount(devicePath, targetPath, opts.FSType, mountOptions)
+	formatOptions := make([]string, 0)
+
+	if opts.FormatOptions != "" {
+		lexer := shell.NewLex('\\')
+		formatOptions, err = lexer.ProcessWords(opts.FormatOptions, shell.EnvsFromSlice([]string{}))
+		if err != nil {
+			return err
+		}
+	} else if opts.FSType == "xfs" {
+		formatOptions = append(formatOptions, "-c")
+		formatOptions = append(formatOptions, fmt.Sprintf("options=%s", XFSDefaultConfigPath))
+	}
```

Comment:
```suggestion
	if opts.FormatOptions != "" {
		lexer := shell.NewLex('\\')
		formatOptions, err = lexer.ProcessWords(opts.FormatOptions, shell.EnvsFromSlice([]string{}))
		if err != nil {
			return err
		}
	}
	if opts.FSType == "xfs" {
		formatOptions = append(formatOptions, "-c")
		formatOptions = append(formatOptions, fmt.Sprintf("options=%s", XFSDefaultConfigPath))
	}
```

Not sure the conditions make sense here?

Reply:
If the user sets formatOptions when using xfs we probably dont want to append `-c options=/usr/share/xfsprogs/mkfs/lts_4.19.cfg`. So either set format options otherwise use fallback if we format with xfs.

Reply:
Makes sens, my bad :)